controllers/nginx/Makefile

```diff
@@ -24,91 +20,20 @@ PKG=k8s.io/ingress/controllers/nginx
 
 ARCH ?= $(shell go env GOARCH)
 GOARCH = ${ARCH}
-DUMB_ARCH = ${ARCH}
```

Comment:
why did you delete a bunch of the make file? Couldn't you just not call these specific rules